.github/workflows/base_container_img.yml

```diff
@@ -0,0 +1,50 @@
+name: Container Img Weekly Build and Release
+
+on:
+  schedule:
+    # This will run every week at 00:00 UTC on Monday
+    - cron: "0 0 * * 1"
```

Comment:
See comment on tag/packages: if we stick with the current scheme, we only need to build this when `bin/.container_tag` changes.